.gitignore

```diff
@@ -22,3 +22,4 @@ public
 npm-debug.log*
 
 .vscode
+.idea
```

Comment:
Why did you add this?

Reply:
Its the webstorm project folder, similar to `.vscode` above.